src/llmcompressor/transformers/compression/quantization_format.py

```diff
@@ -37,7 +36,8 @@ def infer_quantization_format(
     if save_compressed:
         weight_args, input_args = _get_unique_quant_args(model)
         is_24_structure = (
-            sparsity_config and sparsity_config.sparsity_structure == "2:4"
+            SparsityStructure(sparsity_structure).value
+            == SparsityStructure.TWO_FOUR.value
```

Comment:
It seems like we've only enabled this to save using the marlin-24 compressor if the model follows 2:4 sparsity?

Reply:
nit: can we not compare the enums directly without `.value`?

Reply:
@kylesayrs accepted

Reply:
@dsikka let's sync offline